### PR TITLE
579 error handling graphql

### PR DESCRIPTION
**Pull request by DutchBen**
Handles the situation where an error occurs when retrieving data from graphql or during form rendering for example when the eduVPN is inactive

### CODE REVIEW

File: packages/orchestrator-ui-components/src/pages/processes/WfoProcessDetailPage.tsx [hunk 1]
```diff
@@ -41,6 +42,13 @@ export const WfoProcessDetailPage = ({
         fetchInterval,
     );
 
+    if (error && isError) {
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/pages/processes/WfoProcessDetailPage.tsx:45]: An explanation might make this more readable

File: packages/orchestrator-ui-components/src/hooks/useQueryWithGraphql.ts [hunk 2]
```diff
@@ -18,21 +21,26 @@ export const useQueryWithGraphql = <U, V extends Variables>(
...
+        Authorization: session ? `Bearer ${session.accessToken}` : '',
+    };
+
     const fetchFromGraphql = async () =>
         // TS-Ignore because queryVars does not seem to be accepted by the client
         // The props in this useQueryWithGraphql-hook ensures queryVars is indeed related to the query
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-ignore
-        await graphQLClient.request(query, queryVars, requestHeaders);
+        await graphQLClient.request<U, V>(query, queryVars, requestHeaders);
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/hooks/useQueryWithGraphql.ts:36]: You might have fixed the ts-ignore here by adding the types `<U,V>`

Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/hooks/useQueryWithGraphql.ts:36]: I notice that all pages using this hook now contains a `if error then console log` block. Maybe move that logic into this hook. That keeps the consumers of this hook more clean

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/hooks/useQueryWithGraphql.ts (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/hooks/useQueryWithGraphql.ts:36]: No. I have tried a lot but wasn't able to remove it.

Comment (DutchBen) [reply packages/orchestrator-ui-components/src/hooks/useQueryWithGraphql.ts:36]: Done

**Pull request merged**